app/mailers/notifier.rb

```diff
@@ -25,15 +25,34 @@ def notify(notification)
 
   def planting_reminder(member)
     @member = member
+    @sitename = ENV['GROWSTUFF_SITE_NAME']
 
-    @plantings = @member.plantings.order(planted_at: :desc).first(5)
-    @harvests = @member.harvests.order(harvested_at: :desc).first(5)
+    @late = []
+    @super_late = []
+    @harvesting = []
+    @others = []
+
+    @member.plantings.active.annual.each do |planting|
+      if planting.finish_is_predicatable?
+        if planting.super_late?
+          @super_late << planting
+        elsif planting.late?
+          @late << planting
+        elsif planting.harvest_time?
+          @harvesting << planting
+        else
+          @others << planting
+        end
+      end
+    end
+
+    @subject = "Your #{Date.today.strftime('%B %Y')} #{@sitename} progress report"
```

Comment:
i've left this as "Date.today", because Time.zone.today would use the server's timezone, which doesn't make much sense.